packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -93,64 +84,207 @@ export interface IPersonaProps extends React.HTMLAttributes<PersonaBase> {
+   * Call to provide customized styling that will layer on top of variant rules
+   */
+  getStyles?: IStyleFunction<IPersonaStyleProps, IPersonaStyles>;
+
+  /**
+   * Optional custom renderer for the primary text.
+   */
+  onRenderPrimaryText?: IRenderFunction<IPersonaProps>;
+
+  /**
+   * Optional custom renderer for the secondary text.
+   */
+  onRenderSecondaryText?: IRenderFunction<IPersonaProps>;
+
+  /**
+   * Optional custom renderer for the tertiary text.
+   */
+  onRenderTertiaryText?: IRenderFunction<IPersonaProps>;
 
   /**
    * Optional custom renderer for the optional text.
    */
   onRenderOptionalText?: IRenderFunction<IPersonaProps>;
+}
 
+export interface IPersonaStyleProps {
   /**
-   * Whether to not render persona details, and just render the persona image/initials.
+   * Theme.
    */
-  hidePersonaDetails?: boolean;
+  theme: ITheme;
 
   /**
-   * Additional CSS class(es) to apply to the Persona
+   * Custom class name.
    */
   className?: string;
 
+  /**
+   * Optional custom persona coin size in pixel.
+   */
+  coinSize?: number;
+
+  /**
+   * Decides the size of the control.
+   * @defaultvalue PersonaSize.size48
+   */
+  size?: PersonaSize;
+
+  /**
+   * Presence of the person to display - will not display presence if undefined.
+   * @defaultvalue PersonaPresence.none
+   */
+  presence?: PersonaPresence;
+
   /*
    * If true, show the secondary text line regardless of the size of the persona
    */
   showSecondaryText?: boolean;
+}
 
+export interface IPersonaStyles {
+  root: IStyle;
+  placeholder: IStyle;
+  details: IStyle;
+  primaryText: IStyle;
+  secondaryText: IStyle;
+  tertiaryText: IStyle;
+  optionalText: IStyle;
+  textContent: IStyle;
+}
+
+export interface IPersonaCoinProps extends IPersonaSharedProps {
   /**
-   * Optional custom persona coin size in pixel.
+   * Gets the component ref.
    */
-  coinSize?: number;
+  componentRef?: (component: IPersonaCoinProps) => void;
 
   /**
-   * Optional HTML element props for Persona coin.
+   * Call to provide customized styling that will layer on top of the variant rules
    */
-  coinProps?: React.HTMLAttributes<HTMLDivElement>;
+  getStyles?: IStyleFunction<IPersonaCoinStyleProps, IPersonaCoinStyles>;
 
   /**
-   * Call to provide customized styling that will layer on top of variant rules
+   * Additional css class to apply to the PersonaCoin
+   * @defaultvalue undefined
    */
-  getStyles?: IStyleFunction<IPersonaStyleProps, IPersonaStyles>;
+  className?: string;
+}
 
+export interface IPersonaCoinStyleProps {
   /**
-   * Theme provided by HOC.
+   * Theme.
    */
-  theme?: ITheme;
+  theme: ITheme;
+
+  /**
+   * Custom class name.
+   */
+  className?: string;
+
+  /**
+   * Decides the size of the control.
+   * @defaultvalue PersonaSize.size48
+   */
+  size?: PersonaSize;
+}
+
+export interface IPersonaCoinStyles {
+  coin: IStyle;
+  imageArea: IStyle;
+  image: IStyle;
+  initials: IStyle;
+  size10NoPresenceIcon: IStyle;
```

Comment:
size10WithNoPresenceIcon size10WithoutPresenceIcon is a more natural sounding name